packages/mui-base/src/Button/Button.test.tsx

```diff
@@ -67,9 +67,7 @@ describe('<Button />', () => {
         const { getByRole } = render(<Button focusableWhenDisabled disabled />);
 
         const button = getByRole('button');
-        act(() => {
-          button.focus();
-        });
+        button.focus();
```

Comment:
Would expect this one to be wrapped in `act`.

Reply:
Whoops! True, not sure why/how I did this. Fixed.